setup.py

```diff
@@ -33,16 +33,27 @@ def get_c_extension():
         compile_args = ['/openmp']
         link_args = None
     elif platform == 'darwin':
-        # OSX - assume brew install llvm
-        brew_prefix_cmd_return = subprocess.run(["brew", "--prefix"],
-                                                stdout=subprocess.PIPE)
-        brew_prefix = brew_prefix_cmd_return.stdout.decode("utf-8").strip()
-        os.environ['CC'] = '{}/opt/llvm/bin/clang'.format(brew_prefix)
-        compile_args = ['-fopenmp']
-        link_args = ['-L{}/opt/llvm/lib'.format(brew_prefix), '-fopenmp']
+        # OSX - if CC not set, assume brew install llvm
+        try:
+            brew_prefix_cmd_return = subprocess.run(["brew", "--prefix"],
+                                                    stdout=subprocess.PIPE)
+            brew_prefix = brew_prefix_cmd_return.stdout.decode("utf-8").strip()
+        except FileNotFoundError:
+            brew_prefix = None
+
+        if brew_prefix and not os.environ.get['CC']:
```

Comment:
Should be `os.environ.get('CC')`

Sorry for all the failed Mac build emails, I thought because of the force-push the Mac wasn't pulling the correct version of the branch but it turns out I was looking at the wrong part of `setup.py` 😅

Reply:
Whoops!